spec/ddtrace/contrib/grape/tracer_spec.rb

```diff
@@ -306,7 +327,7 @@
           expect(run_span.name).to eq('grape.endpoint_run')
           expect(run_span.span_type).to eq('web')
           expect(run_span.service).to eq('grape')
-          expect(run_span.resource).to eq('TestingAPI#hard_failure')
+          expect(run_span.resource).to eq('TestingAPI GET /base/hard_failure')
```

Comment:
What do you think of this resource name? I think it captures the code structure around this endpoint, allowing you unambiguously identify the affected code path, without having distinct parts joined together.
The extended information (HTTP method, full path) are still captured as tags.

Also, please let me know if this does not actually unambiguously identify the endpoint.

```suggestion
          expect(run_span.resource).to eq('TestingAPI::base#hard_failure')
```